### DISCUSSION TITLE
How can I completely delete lazy vim?

### DISCUSSION THREAD

**Discussion by ashtotakoe** [LazyVim/LazyVim#498] [Q&A] [answered]
I want to completely uninstall lazy vim to reinstall it from the ground up

**Comment by dpetka2001** [comment, marked as answer]
Well since you must have cloned the starter repo and worked your way up from there, you should just remove that directory as well as `~/.local/state/nvim` and `~/.local/share/nvim` directories.

**Reply by ashtotakoe** [reply]
Thank you, it helped!
